app/models/customization_template.rb

```diff
@@ -5,11 +5,15 @@ class CustomizationTemplate < ApplicationRecord
   has_many   :pxe_images, :through => :pxe_image_type
 
   validates :pxe_image_type, :presence => true, :unless => :system?
-  validates :name,           :unique_within_region => true
+  validates :name, :uniqueness => {:scope => :pxe_image_type }, :if => :not_seeding?
 
   scope :with_pxe_image_type_id, ->(pxe_image_type_id) { where(:pxe_image_type_id => pxe_image_type_id) }
   scope :with_system,            ->(bool = true)       { where(:system => bool) }
 
+  def not_seeding?
```

Comment:
Can we make the method name not a negative? :wink:

Reply:
Sure, give me a good name for it.  

I honestly have no idea why we're seeding customization templates with no pxe_image_type if pxe_image_type is a required for validation of a customization template.  This whole thing would be so much easier if seeding behaved properly.  @bdunne do you know why seeding wants to be different?

Reply:
Ignoring the validity of creating templates without a pxe_image_type for a second I'd be fine with just `seeded_template?`. I really just wanted to drop the `not_` from the method.

Reply:
@jrafanie The system images are not usable by customers.  They're just there as examples that the user can copy.  If you want, you can change seeding so that they get seeded into the system customization templates PxeImageType.